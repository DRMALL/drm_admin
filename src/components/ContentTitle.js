

import React from 'react'
import PageTip from './PageTip'
import ContentLink from './ContentLink'


const ContentTitle = ({ head }) => (
  <div className='content-head' >
    <div className='content-head-left' >
      <PageTip tip={ head.title } />
    </div>
    <ContentLink tip = { head.tip } path={ head.path } />
  </div>
  )

export default ContentTitle

