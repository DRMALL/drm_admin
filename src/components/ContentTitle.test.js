

import React from 'react'
import ContentTitle from './ContentTitle'
import renderer from 'react-test-renderer'
// import { render, shallow,mount } from 'enzyme'
// import toJson from 'enzyme-to-json'


// it('contentTitle render test', () => {
//   const head = {}
//   head.title = '故障排除'
//   head.path = '/trouble/new'
//   head.tip = '创建新信息'
//   const tree = shallow(<ContentTitle head={ head } />)
//   expect(toJson(tree)).toMatchSnapshot()
// })

it('contentTitle render test', () =>{
  const head = {}
  head.title = '故障排除'
  head.path = '/trouble/new'
  head.tip = '创建新信息'
  const tree = renderer.create(<ContentTitle head={ head } />)
  expect(tree).toMatchSnapshot()
})

