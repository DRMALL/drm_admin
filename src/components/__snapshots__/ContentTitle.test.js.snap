// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`contentTitle render test 1`] = `
<div
  className="content-head"
>
  <div
    className="content-head-left"
  >
    <div
      className="page-tip"
    >
      故障排除
    </div>
  </div>
  <a
    className="content-link"
    onClick={[Function]}
    style={Object {}}
  >
    <div
      className="content-link-block"
    >
      <i
        className="anticon anticon-plus"
      />
    </div>
    <div
      className="content-link-text"
    >
      创建新信息
    </div>
  </a>
</div>
`;
